lib/entities/quote/DutchQuote.ts

```diff
@@ -70,7 +71,13 @@ export class DutchQuote implements IQuote {
   public static amountInImprovementExactOut = BigNumber.from(9999);
 
   // build a dutch quote from an RFQ response
-  public static fromResponseBody(request: DutchRequest, body: DutchQuoteJSON, nonce?: string): DutchQuote {
+  public static fromResponseBody(
+    request: DutchRequest,
+    body: DutchQuoteJSON,
+    nonce?: string,
+    portionBips?: number,
+    portionRecipient?: string
```

Comment:
nit can we colocate the fee info with an object type? think `type PortionData = { portionBips, portionRecipient }` would make this more readable, and could check for nullness on a single field rather than both. this is also more expressive of the fact that they  are mutually required